sphinx/writers/latex.py

```diff
@@ -572,6 +572,19 @@ def __init__(self, document, builder):
         else:
             self.elements['date'] = format_date(builder.config.today_fmt or _('%b %d, %Y'),  # type: ignore  # NOQA
                                                 language=builder.config.language)
+
+        if builder.config.numfig:
+            self.numfig_secnum_depth = builder.config.numfig_secnum_depth
+            if self.numfig_secnum_depth > 0:  # default is 1
+                self.numfig_secnum_depth += self.top_sectionlevel - 1
```

Comment:
this will need modification once PR #4316 is merged as it modifies self.top_sectionlevel for howto documents